python/ray/autoscaler/commands.py

```diff
@@ -86,30 +91,122 @@ def request_resources(num_cpus=None, bundles=None):
 
 def create_or_update_cluster(config_file, override_min_workers,
                              override_max_workers, no_restart, restart_only,
-                             yes, override_cluster_name, no_config_cache):
+                             yes, override_cluster_name, no_config_cache,
+                             log_old_style, log_color, verbose):
     """Create or updates an autoscaling Ray cluster from a config json."""
-    config = yaml.safe_load(open(config_file).read())
-    if override_min_workers is not None:
-        config["min_workers"] = override_min_workers
-    if override_max_workers is not None:
-        config["max_workers"] = override_max_workers
-    if override_cluster_name is not None:
-        config["cluster_name"] = override_cluster_name
+    cli_logger.old_style = log_old_style
+    cli_logger.color_mode = log_color
+    cli_logger.verbosity = verbose
+    cli_logger.dump_command_output = verbose == 3  # todo: add a separate flag?
```

Comment:
fixup?

Reply:
going to set this to `True` for now since it is actually unused (this was for the command output handling part of this PR that got moved into a future PR)